queries.sql

```diff
@@ -99,3 +99,22 @@ JOIN animals ON  owners.id = animals.owner_id
 GROUP BY full_name
 ORDER BY COUNT(name) DESC;
 
+/*join table queries*/
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230728608-a8982c42-2da4-487e-9fa8-1fd00d5e8574.png)

**[OPTIONAL]**

- [ ] It would be very great if you make all your commit messages more meaningful and capitalize because I saw your commit are not capitalized 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍